soc/arc/snps_nsim/CMakeLists.txt

```diff
@@ -7,6 +10,34 @@ zephyr_compile_options_ifdef(CONFIG_FPU -mfpu=fpud_all)
 else()
 zephyr_compile_options_ifdef(CONFIG_FPU -mfpu=fpuda_all)
 endif()
+else()
+# MWDT compiler options
+zephyr_compile_options_ifdef(CONFIG_SOC_NSIM_EM -arcv2em -core3 -Xdiv_rem=radix2
+			     -Xmpy_option=mpyd -Xbitscan -Xswap -Xbarrel_shifter
```

Comment:
With MetaWare, we could use a tcf file instead of these.

Reply:
Moreover the .tcf file may includes the properties for nSIM and MDB as well. So, I propose to have it as a separate enhancement - to switch all metaware tools (compiler, simulator, debugger) to use tcf.

Reply:
Agreed, this is a separate topic and should extend to other tools (maybe even including ARC gnu).